pkg/api/ldap.go

```diff
@@ -85,6 +85,7 @@ func (lc *LDAPClient) Authenticate(username, password string) (bool, map[string]
 	if lc.BindDN != "" && lc.BindPassword != "" {
 		err := lc.Conn.Bind(lc.BindDN, lc.BindPassword)
 		if err != nil {
+			lc.Conn = nil
```

Comment:
Is there any kind of lc.Conn.Close() you need to do?

Reply:
Likely to fail in this particular case, but we can attempt it.

Reply:
Even if part of it fails, it still may release some resources, so worth doing I think.